modules/reindex/src/main/java/org/elasticsearch/index/reindex/AbstractAsyncBulkByScrollAction.java

```diff
@@ -19,6 +19,18 @@
 
 package org.elasticsearch.index.reindex;
 
+import java.util.ArrayList;
```

Comment:
It all comes from the gradle config. Maybe it comes from going between 2.3 and master. I can see about it. Until we actually check it with tools stuff like this will leak out because most of us are import blind.